src/services/accounts/AccountsStakingInfoService.ts

```diff
@@ -66,6 +67,64 @@ export class AccountsStakingInfoService extends AbstractService {
 			);
 		}
 
+		let claimedRewardsEras: u32[] = [];
+		let claimedRewards: IEraStatus[] = [];
+
+		if ((stakingLedger as unknown as IStakingLedger)?.lastReward) {
+			const e = (stakingLedger as unknown as IStakingLedger)?.lastReward?.unwrap().toNumber();
+			if (e) {
+				claimedRewards.push({ era: e, status: 'claimed' });
+			}
+		}
+
+		if (stakingLedger?.legacyClaimedRewards) {
+			claimedRewardsEras = stakingLedger?.legacyClaimedRewards;
+		} else {
+			claimedRewardsEras = (stakingLedger as unknown as IStakingLedger)?.claimedRewards as Vec<u32>;
+		}
+		if (claimedRewardsEras) {
+			claimedRewards = claimedRewardsEras.map((element) => ({
+				era: element.toNumber(),
+				status: 'claimed',
+			}));
+		}
+		if (historicApi.query.staking?.claimedRewards) {
+			const currentEraMaybeOption = await historicApi.query.staking.currentEra();
+			const currentEra = currentEraMaybeOption.unwrap().toNumber();
```

Comment:
same question as above

Reply:
You are right & thank you! In the same [commit](https://github.com/paritytech/substrate-api-sidecar/pull/1445/commits/753f3eefde99a1fa750a972ba395ee53017131e5) I added a check for `isNone` before unwrapping.